### PR TITLE
Update getting-started-with-swashbuckle.md

### LINKED ISSUE #31651: Fix limitations
### Description

[Enter feedback here]

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-7.0&viewFallbackFrom=aspnetcore-8.0&branch=pr-en-us-31647&tabs=visual-studio

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/getting-started-with-swashbuckle.md

### Document ID

59222460-a971-e76e-4fe9-23d4d7d768ee

### Article author

@zuckerthoben

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #31651

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/getting-started-with-swashbuckle.md](https://github.com/dotnet/AspNetCore.Docs/blob/291e49048deb35ce76926437dee0aa9ba5c02fde/aspnetcore/tutorials/getting-started-with-swashbuckle.md) | [Get started with Swashbuckle and ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?branch=pr-en-us-31647) |

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/getting-started-with-swashbuckle.md](https://github.com/dotnet/AspNetCore.Docs/blob/300c33022a82296d066e31d714defb8cbddefa1b/aspnetcore/tutorials/getting-started-with-swashbuckle.md) | [Get started with Swashbuckle and ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?branch=pr-en-us-31647) |

### CODE REVIEW

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ title: Get started with Swashbuckle and ASP.NET Core
...
+monikerRange: '>= aspnetcore-3.1'
 ms.custom: mvc
 ms.date: 11/09/2021
 uid: tutorials/get-started-with-swashbuckle
 ---
 # Get started with Swashbuckle and ASP.NET Core
 
 > [!NOTE]
-> Build-time OpenAPI document generation with Swashbuckle is not supported in .NET 8 and later. For a supported alternative, see <xref:tutorials/web-api-help-pages-using-swagger>.
+> Build-time OpenAPI document generation with Swashbuckle is not supported in .NET 8 and later. For a supported  build-time  alternative, see <xref:tutorials/web-api-help-pages-using-swagger>. Run-time support is not impacted.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/getting-started-with-swashbuckle.md:14]: @captainsafia The original was off putting. I'm hoping to tone it down, closer to the actual limitations.

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md [hunk 2]
```diff
@@ -3,15 +3,15 @@ title: Get started with Swashbuckle and ASP.NET Core
 author: zuckerthoben
 description: Learn how to add Swashbuckle to your ASP.NET Core web API project to integrate the Swagger UI.
 ms.author: scaddie
-monikerRange: '>= aspnetcore-3.1 < aspnetcore-8.0'
+monikerRange: '>= aspnetcore-3.1'
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/getting-started-with-swashbuckle.md:6]: @captainsafia this change removes redirect to .NET 7

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md (hunk 1 above)
Comment (captainsafia) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md:14]: This change looks good!

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md [hunk 3]
```diff
@@ -3,15 +3,15 @@ title: Get started with Swashbuckle and ASP.NET Core
...
 ms.custom: mvc
 ms.date: 11/09/2021
 uid: tutorials/get-started-with-swashbuckle
 ---
 # Get started with Swashbuckle and ASP.NET Core
 
 > [!NOTE]
```
Comment (captainsafia) [inline aspnetcore/tutorials/getting-started-with-swashbuckle.md:13]: Should this note be configured to only display for net8.0 and later?

**Review by captainsafia — APPROVED** [review]

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md (hunk 1 above)
Comment (tdykstra) [inline aspnetcore/tutorials/getting-started-with-swashbuckle.md:14]:
```suggestion
> Build-time OpenAPI document generation with Swashbuckle isn't supported in .NET 8 and later. For a supported build-time alternative, see <xref:tutorials/web-api-help-pages-using-swagger>. Swashbuckle still supports run-time document generation.
```

**Review by tdykstra — APPROVED** [review]
Looks good, a couple suggestions.

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md (hunk 1 above)
Comment (captainsafia) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md:16]: > Swashbuckle still supports run-time document generation.

The phrasing here feels off to me. Maybe "Run-time document generation is still supported in .NET 8."

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md (hunk 3 above)
Comment (Rick-Anderson) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md:15]: > Should this note be configured to only display for net8.0 and later?

Fixed

**Pull request merged**